mkdocs.yml

```diff
@@ -82,14 +84,29 @@ plugins:
         'about.md': 'participate.md'
   - external-markdown
   - i18n:
-      docs_structure: folder
+      docs_structure: suffix
```

Comment:
We will use `folder` as the structure, we already decided on that. It's a mess otherwise with tons of files with the same name.

Reply:
Done ✅